src/pages/my_page/user_profile/MyFieldAndBackground.tsx

```diff
@@ -0,0 +1,138 @@
+import { LoadingButton } from '@mui/lab';
+import { Box, TextField, Typography } from '@mui/material';
+import { useMutation, useQuery } from '@tanstack/react-query';
+import { Field, FieldProps, Form, Formik, FormikProps } from 'formik';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchPerson, updateCristinPerson } from '../../../api/cristinApi';
+import { BackgroundDiv } from '../../../components/styled/Wrappers';
+import { setNotification } from '../../../redux/notificationSlice';
+import { RootState } from '../../../redux/store';
+import { FlatCristinPerson } from '../../../types/user.types';
+import { ResearchProfilePanel } from './ResearchProfilePanel';
+
+type CristinPersonFormData = Pick<FlatCristinPerson, 'background'>;
+
+export const MyFieldAndBackground = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const user = useSelector((store: RootState) => store.user)!;
+  const personId = user?.cristinId ?? '';
+
+  const personQuery = useQuery({
+    queryKey: [personId],
+    queryFn: () => fetchPerson(personId),
+    meta: { message: t('feedback.error.get_person'), variant: 'error' },
+  });
+
+  const person = personQuery.data;
+  const personBackground = person?.background ?? {};
+
+  const initialValues: CristinPersonFormData = {
+    background: {
+      no: personBackground.no ? personBackground.no : '',
+      en: personBackground.en ? personBackground.en : '',
+    },
+  };
+
+  const updatePerson = useMutation({
+    mutationFn: async (values: CristinPersonFormData) => {
+      if (user.cristinId) {
```

Comment:
Har allerede definert `personId` lenger opp, som jeg antar ble gjort for å brukes her også?

```suggestion
      if (personId) {
```